evadb/third_party/databases/clickhouse/clickhouse_handler.py

```diff
@@ -0,0 +1,185 @@
+# coding=utf-8
+# Copyright 2018-2023 EvaDB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pandas as pd
+from sqlalchemy import create_engine
+# from clickhouse_sqlalchemy.drivers.base import ClickHouseDialect
```

Comment:
Please add a requirements.txt for `clickhouse_sqlalchemy`. And remove commented code.